### ISSUE TITLE
[BUG] npm ERR! integrity checksum failed when using sha512

### ISSUE THREAD

**Issue by ctung** [npm/cli#3059]
I'm just trying to do a fresh install of nodejs, npm & angular/cli

### Current Behavior:
Fresh install of nodejs and npm
npm i -g @angular/cli

FIrst I get " UNABLE_TO_GET_ISSUER_CERT_LOCALLY " errors.
I work around that by setting:
npm config set registry=http://registry.npmjs.org/

Now I get a bunch of "tarball data... seems to be corrupted" errors followed by "integrity checksum failed when using sha512"

log file:
[2021-04-11T04_32_26_844Z-debug.log](https://github.com/npm/cli/files/6291194/2021-04-11T04_32_26_844Z-debug.log)

The first indication of a problem, is that its complaining about using stale data from registry.npmjs.org
```
249 warn registry Unexpected warning for http://registry.npmjs.org/: Miscellaneous Warning EINTEGRITY: sha512-yM9uuShT/BVjyT7UT/j5NDAsu3flTdpUnWSwSMc1pJ43NfBf6VHuPH1FVHudKHM5iKKA7bxAYVpmoNz3ncgFIA== integrity checksum failed when using sha512: wanted sha512-yM9uuShT/BVjyT7UT/j5NDAsu3flTdpUnWSwSMc1pJ43NfBf6VHuPH1FVHudKHM5iKKA7bxAYVpmoNz3ncgFIA== but got sha512-FTBIXP6GGNllnYqPlb/J9pRXO3bX32QEDUaX9QClJM2s/pVHEqeGCkvCOzvIWg6dFsU1yfD0KWyqOrzxGg/3QQ==. (15643 bytes)
250 warn registry Using stale data from http://registry.npmjs.org/ due to a request error during revalidation.
```

I'm able to access http://registry.npmjs.org and I don't think I'm using any proxies.

If I try to run npm cache verify, it removes a handful of corrupted content each time:
```
PS C:\Users\chuck> npm cache verify
Cache verified and compressed (~\AppData\Local\npm-cache\_cacache)
Content verified: 459 (16331257 bytes)
Corrupted content removed: 13
Content garbage-collected: 14 (2255149 bytes)
Missing content: 19
Index entries: 669
Finished in 1.739s
PS C:\Users\chuck> npm cache verify
Cache verified and compressed (~\AppData\Local\npm-cache\_cacache)
Content verified: 444 (15944836 bytes)
Corrupted content removed: 15
Content garbage-collected: 15 (386421 bytes)
Missing content: 25
Index entries: 644
Finished in 1.568s
PS C:\Users\chuck> npm cache verify
Cache verified and compressed (~\AppData\Local\npm-cache\_cacache)
Content verified: 431 (15660557 bytes)
Corrupted content removed: 13
Content garbage-collected: 13 (284279 bytes)
Missing content: 20
Index entries: 624
Finished in 1.502s
PS C:\Users\chuck> npm cache verify
Cache verified and compressed (~\AppData\Local\npm-cache\_cacache)
Content verified: 424 (15601138 bytes)
Corrupted content removed: 7
Content garbage-collected: 7 (59419 bytes)
Missing content: 12
Index entries: 612
Finished in 1.47s
```

### Expected Behavior:
haven't done this in a while but it used to just work...
I've tried:
- npm cache verify
- npm cache clean --force
- npm set strict-ssl=false
- can't delete package-lock.json because I'm not far enough along to even have a project
- delete AppData/Local/npm-cache
- delete AppData/Roaming/npm-cache
- delete AppData/Roaming/npm
- my home internet connection
- VPN to my work through Boston
- VPN to my work through Atlanta
- both nodejs current and LTS
- npm config set cafile "/path/to/verisign_universal_root_cert.pem"

### Steps To Reproduce:
uninstall nodejs
delete AppData/Local/npm-cache
delete AppData/Roaming/npm
reinstall nodejs (either current or LTS)
npm i @angular/cli

### Environment:
- Windows 10
- Node: v15.14.0
- npm: 7.9.0

**Comment by ctung** [comment]
This is not a fix for npm, but for other people stuck in the same boat as me, I worked around npm by using yarn.
```
npm install --global yarn
yard global add @angular/cli
ng config -g cli.packageManager yarn
ng new my-project
```
Even with yarn, I ran into "Integrity check failed" during my "ng new".  However I was able to workaround with:
```
cd my-project
yarn cache clean
yarn --update-checksums
```

**Comment by ctung** [comment]
Further progress...
I changed windows to safe mode, to see if I was loading something that was causing the problem.
Turns out if you switch to safe mode without a windows password (not pin), you brick your Windows installation.
So after re-installing windows... no more integrity check issues.

I suspect there was something on my computer that was either acting as a proxy, or altering shaping the files as they got downloaded, causing all my integrity checks to fail.

Not that its a great solution, but re-installing windows, fixed my issues...
